### PR TITLE
85 enable more hurricane demand type and units

### PR DESCRIPTION
**Pull request by longshuicy**
It's too hard to test this PR. I'm wondering if i can deploy this on incore-tst or incore-dev

### CODE REVIEW

**Comment by ywkim312** [comment]
This PR has been updated in incore-tst. @longshuicy Is there a code or notebook or api endpoint url that I can test this?

**Comment by longshuicy** [comment]
> This PR has been deployed in incore-tst. @longshuicy Is there a code or notebook or api endpoint url that I can test this?

you can try post to
`https://incore-tst.ncsa.illinois.edu/hazard/api/hurricanes/5fa5a83c7e5cdf51ebf1adae/values`
image

The body is form-data. Point:  `[{"demands": ["windVelocity", "waveDirection", "waterVelocity", "inundationDuration"], "units": ["m/s", "deg", "m/s", "hr"], "loc": "29.27627444,-94.83624111"}]`

**Review by ywkim312 — APPROVED** [review]
I don't know the numbers in the code if they are correct or not. However, the code looks okay to me and endpoint test using tst worked well.

**Comment by ywkim312** [comment]
@longshuicy This is what I got. I am only concerning about the hazard values that are -9999. Please confirm is the values are correct 
```
[
    {
        "hazardValues": [
            -9999.2,
            -9999.2
        ],
        "demands": [
            "3s",
            "60s"
        ],
        "units": [
            "mph",
            "mph"
        ],
        "loc": "28.0, -81.0"
    }
]
```

**Comment by ywkim312** [comment]
This is what I get 
```
[
    {
        "hazardValues": [
            45.666875,
            281.61859,
            null,
            9.4474448
        ],
        "demands": [
            "windVelocity",
            "waveDirection",
            "waterVelocity",
            "inundationDuration"
        ],
        "units": [
            "m/s",
            "deg",
            "m/s",
            "hr"
        ],
        "loc": "29.27627444, -94.83624111"
    }
]
```

**Comment by longshuicy** [comment]
@ywkim312  could you also help me deploy this on incore-dev whenever you have chance? Or just merge it to develop :p

**Comment by ywkim312** [comment]
> @ywkim312 could you also help me deploy this on incore-dev whenever you have chance? Or just merge it to develop :p

dev needs to be figured out with new traefik2. I will deploy as soon as we resolve this on dev and will let you know.

**Comment by longshuicy** [comment]
> 

no hurry. Thanks!

File: server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/models/hurricane/utils/HurricaneUtil.java [hunk 1]
```diff
@@ -27,9 +27,51 @@ public class HurricaneUtil {
...
+    private static final String units_mph = "mph";
+
+    // Degree
+    private static final String degree = "deg";
+    private static final String radian = "rad";
 
     public static final List<String> distanceUnits = Arrays.asList(units_cm, units_m, units_in, units_ft);
-    public static final List<String> durationUnits = Arrays.asList(units_hr, units_min);
+    public static final List<String> durationUnits = Arrays.asList(units_hr, units_min, units_s);
+    public static final List<String> speedUnits = Arrays.asList(units_m_per_s, units_in_per_s, units_ft_per_s, units_mph);
```
Comment (navarroc) [inline server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/models/hurricane/utils/HurricaneUtil.java:45]: I think you need to add units_m_per_sec here.

Comment (ywkim312) [reply server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/models/hurricane/utils/HurricaneUtil.java:45]: let me know when this code get updated. I will deploy to tst

Comment (longshuicy) [reply server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/models/hurricane/utils/HurricaneUtil.java:45]: sorry i missed that!

**Review by navarroc — APPROVED** [review]

**Pull request merged**